pages/content/amp-dev/documentation/guides-and-tutorials/index.email.html

```diff
@@ -8,114 +8,199 @@
   path: /{locale}/documentation/guides-and-tutorials.email.html
 $order: 1
 $hidden: true
+
+flyout:
+  description: Get started with AMP
+  icon: guides-documents
+cms: !g.yaml /content/shared/data/cms.yaml
```

Comment:
Missed this earlier: this can be removed from all of the four pages as we don't need to render the CMSes anymore on those pages.

Reply:
Done - removed from all four pages